### ISSUE TITLE
Can we buy one little slice frame of the copy of the Archive Reel?

### ISSUE THREAD

**Issue by SaekiRaku** [github/archive-program#45]
* How much does it cost to produce a frame of the reel?
* Can we buy one slice of that as a souvenir? Does this have a plan?
* Can the souvenir record a short `Hello World` programme without compression? So that the buyer can try to use `a source of illumination and some kind of magnifier` to read the code directly, kind of fun.
* Is the craftsmanship have a private patent? Can other factories imitation this?

**Comment by boxlab** [comment]
Good idea 👍

**Comment by DrBeta** [comment]
lol I would love a backup of my terrible code  😂😂😂

**Comment by DrBeta** [comment]
Could we make the souvenir include private repos?

**Comment by rezendi** [comment]
This is a fun idea but, alas, logistically challenging. After some deliberation, we're not going to move forward with in the foreseeable future, but may well eventually reconsider. We'll leave this issue open though -

**Comment by craSH** [comment]
@rezendi is the code used to produce the reel images available? I would expect it to be open source, if not also included in the archive but I have not been able to find that.

Are there sample images (e.g. SVG or PNG) of any reel prints available for the public to see, and to attempt to verify the accuracy of the Vault guide and the Tech Tree?

**Comment by homelyseven250** [comment]
Yes - a piece of human history, storing open source code is not open source itself!

**Comment by PalinuroSec** [comment]
i personally bought some rolls of black and white analog film, prepared some nice images with both qr code and human readable text and shot them with an analog camera.

i own everything needed to develop the film at home on myself, but analog photography development kits are available online for those of you brave enough who want to experiment with film photography.

my experiment was successful and now i have my little roll of developed silver halide film with some code impressed into it.

of course mine is not the same standard used at the AWA facility, and the photographic film does not use the same technology used by piql, so my roll is neither as durable nor properly preserved, but the satisfaction to own some data stored in such an unusual way was over 9000!

**Comment by DrBeta** [comment]
Github should put this in their march store

**Comment by ashu-cybertron** [comment]
this ia great initiative

**Comment by juliannojungle** [comment]
> @rezendi is the code used to produce the reel images available? I would expect it to be open source, if not also included in the archive but I have not been able to find that.

What a great idea! If the code to reproduce the reel were available, we could DIY our own "reel" by laser engraving our code in some simpler material (or something like that). Even "3D printing code" could be fun, since we probably would never get access to the real deal.

**Comment by harrypm** [comment]
Just thought I would point out ADOX CMS20 II is open market 35mm Microfise film, with standard digital to 35mm burning systems that do 8k or 36mp of digital to film data we can store PCM block style data or BLOCKS style rosseta leaders easily.
